sdk/identity/identity/src/credentials/chainedTokenCredential.ts

```diff
@@ -48,8 +48,16 @@ export class ChainedTokenCredential implements TokenCredential {
    *                `TokenCredential` implementation might make.
    */
   async getToken(scopes: string | string[], options: GetTokenOptions = {}): Promise<AccessToken> {
+    const token = (await this.getTokenInternal(scopes, options)).token;
+    return token;
```

Comment:
style nit, I think this is a little more readable

```suggestion
    const { token } = await this.getTokenInternal(scopes, options);
    return token;
```